### ISSUE TITLE
Solve CherryPick

### ISSUE THREAD

**Issue by cswinter** [entity-neural-network/incubator#48]
Achieve > 0.99 reward on the `CherryPick` env with default env-kwargs:

```
poetry run python enn_ppo/enn_ppo/train.py --gym-id=CherryPick --track --env-kwargs='{"num_cherries": 32}'
```

**Comment by cswinter** [comment]
This is blocked on https://github.com/entity-neural-network/incubator/issues/19

**Comment by cswinter** [comment]
No longer blocked.

Partial solution with 4 cherries on : `poetry run python enn_ppo/enn_ppo/train.py --gym-id=CherryPick --num-envs=64 --num-steps=16 --track --total-timesteps=100000 --env-kwargs='{"num_cherries": 4}' --learning-rate=0.003`
[W&B](https://wandb.ai/entity-neural-network/enn-ppo/runs/3qpoed8a)

**Comment by cswinter** [comment]
Solved on 1dac205 by `poetry run python enn_ppo/enn_ppo/train.py --gym-id=CherryPick --track --env-kwargs='{"num_cherries": 32}' --total-timesteps=100000 --learning-rate=0.005 --num-envs=128 --processes=4 --d-model=256 --n-layer=2 --num-steps=64 --gamma=0.99 --num-minibatches=4 --ent-coef=0.001 --vf-coef=0.25`.
[W&B](https://wandb.ai/entity-neural-network/enn-ppo/runs/3d2a2nkz)

One slightly concerning observation, sometimes after the policy has achieved perfect play, there is a massive grad spike that destroys the policy. My best guess is that this is caused by some numerical issue once the policy becomes extremely confident in the right choice.
